src/pages/PeoplePage.tsx

```
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { Loader } from '../components/Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    setError(false);
+
+    getPeople()
```

This file relies on the getPeople() API helper (line 17). Ensure that helper requests the exact required URL `https://mate-academy.github.io/react_people-table/api/people.json` and that it returns `Promise<Person[]>` and rejects on network errors. If getPeople swallows errors, the error UI here will not be triggered.